R/packages.R

```diff
@@ -1,25 +1,43 @@
 #' Using Packages with rsconnect
 #'
-#' @description
-#' Deployed applications can depend on any package available on CRAN as well as any package hosted in a public \href{https://github.com/}{GitHub} repository.
+#' @description Deployed applications can depend on any package available on
```

Comment:
These comments were adjusted using the Code->Reflow Comment menu option in RStudio.